src/ApparentHorizons/ComputeItems.hpp

```diff
@@ -0,0 +1,74 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+/// \cond
+class DataVector;
+/// \endcond
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Christoffel.hpp"
+#include "PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp"
+#include "PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp"
+#include "PointwiseFunctions/GeneralRelativity/IndexManipulation.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace ah {
+namespace Tags {
+// These ComputeItems are different from those used in
+// GeneralizedHarmonic evolution because these live only on the
+// intrp::Actions::ApparentHorizon DataBox, not in the volume
+// DataBox.  And these ComputeItems can do fewer allocations than the
+// volume ones, because (for example) Lapse, SpaceTimeNormalVector,
+// etc.  can be inlined instead of being allocated as a separate
+// ComputeItem.
+template <size_t Dim, typename Frame>
```

Comment:
I think you should add documentation for these tags. Perhaps just wrap them in `@{}` and use the comment that's already there, with a reference to the GH compute items.

Reply:
Moved documentation to doxygen.  The GH compute items are not written yet, so can't reference them.